pyodide-build/setup.cfg

```diff
@@ -29,11 +29,17 @@ install_requires =
     build==0.7.0
     virtualenv
     pydantic>=1.10.2
+    pyodide-cli>=0.2.0
+    typer
```

Comment:
You can probably remove the typer dependency. It's up to pyodide-cli to define the version of typer it need.